deploy/helm/sumologic/Chart.yaml

```diff
@@ -37,6 +37,6 @@ dependencies:
     repository: https://sumologic.github.io/tailing-sidecar
     condition: tailing-sidecar-operator.enabled
   - name: opentelemetry-operator
-    version: 0.18.3
+    version: 0.24.0
```

Comment:
please update Opentelemtery Operator version in support matrix here: https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix